malinka.el

```diff
@@ -1,4 +1,4 @@
-;;; malinka.el --- A C/C++ project configuration package for Emacs
+;;; malinka.el --- A C/C++ project configuration package for Emacs -*- lexical-binding: t; -*-
```

Comment:
I made a change to that in the other pull request. (Using push instead of add-to-list)

Dynamic binding was making it a little tricky to debug the callbacks (the names kept colliding together).

https://www.reddit.com/r/emacs/comments/5j9qgz/lexicalbinding_t_vs_lexicallet/